documentation/modules/upgrading/proc-upgrade-cli-tool-files.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// assembly-upgrade-resources.adoc
+
+[id='proc-upgrade-cli-tool-files-{context}']
+= Converting custom resources configuration files using the API conversion tool
+
+[role="_abstract"]
+This procedure describes how to use the API conversion tool to convert YAML files describing the configuration for Strimzi custom resources into a format applicable to `{ApiVersion}`.
+To do so, you use the `convert-file` (`cf`) command.
+
+The `convert-file` command can convert YAML files containing multiple documents.
+For a multi-document YAML file, all the Strimzi custom resources it contains are converted.
+Any non-Strimzi Kubernetes resources are replicated unmodified in the converted output file.
+
+After you have converted the YAML file, you can apply the configuration to update the custom resource in the cluster.
```

Comment:
Again, not a native English speaker ... but is the _you can apply_ sufficiently clear that they have to do it? Because if they don't it is as if they haven't done anything.

Reply:
Replaced with _must_ to be definite, though we do say _conversion is only complete..._ at the end of the paragraph